src/main/java/com/synopsys/integration/detect/lifecycle/run/operation/OperationRunner.java

```diff
@@ -428,15 +438,29 @@ public JsonObject createContainerScanImageMetadata(UUID scanId, NameVersion proj
     // Generic method to POST a file to /api/storage/containers endpoint of storage service
     public Response uploadFileToStorageService(BlackDuckRunData blackDuckRunData, String storageServiceEndpoint, File payloadFile, String postContentType, String operationName)
         throws OperationException {
+        return uploadFileToStorageServiceWithHeaders(blackDuckRunData, storageServiceEndpoint, payloadFile, postContentType, operationName, null);
+    }
+    
+    public Response uploadFileToStorageServiceWithHeaders(BlackDuckRunData blackDuckRunData, String storageServiceEndpoint, File fileToUpload, String postContentType, String operationName, Map<String, String> headers) 
```

Comment:
I changed the variables to align. For now I'm thinking I'll leave the two methods. I guess this comes down to personal preference somewhat. I did it this way to not disrupt the caller, which admittedly isn't a big deal since the caller and code are in the same code base. Still, I have a slight preference for the straight upload method and the one that does a bit more (even though they call into one another). For now I'll leave this open and we can see how others feel when we open the MR up.